src/gui/tray/UserModel.cpp

```diff
@@ -486,6 +487,9 @@ void User::slotItemCompleted(const QString &folder, const SyncFileItemPtr &item)
             _activityModel->addIgnoredFileToList(activity);
         } else {
             // add 'protocol error' to activity list
+            if (item->_status == SyncFileItem::Status::FileNameInvalid) {
+                emit guiLog(item->_file, activity._subject);
```

Comment:
@FlexW Not sure, but, I think this should be moved into `void User::slotBuildNotificationDisplay(const ActivityList &list)` as the other `emit guiLog()` is already there. Just to keep this part clean of redundant code :)

Reply:
Yep, agreed with this. If that's doable it's better.